distribution/assembly/bin.xml

```diff
@@ -75,13 +90,27 @@
                 <include>commons-pool:commons-pool:jar</include>
                 <include>commons-io:commons-io:jar</include>
                 <include>com.lmax:disruptor:jar</include>
+
+                <include>com.google.code.findbugs:jsr305:jar</include>
+                <include>com.google.guava:guava:jar</include>
+                <include>commons-codec:commons-codec:jar</include>
+                <include>javax.xml.stream:stax-api:jar</include>
+                <include>joda-time:joda-time:jar</include>
+                <include>org.apache.httpcomponents:httpclient:jar</include>
+                <include>org.apache.httpcomponents:httpcore:jar</include>
+                <include>org.bouncycastle:bcprov-jdk15on:jar</include>
+                <include>org.codehaus.woodstox:stax2-api:jar</include>
+                <include>org.codehaus.woodstox:woodstox-core-asl:jar</include>
+                <include>org.cryptacular:cryptacular:jar</include>
```

Comment:
These dependencies are transitive dependencies of OpenSAML 3 dependencies. The reason behind not including these dependencies within shaded OpenSAML 3 artifacts themselves is because multiple OpenSAML 3 dependencies require these.
On one or two instances I have included transitive dependencies which are specific to only one OpenSAML 3 dependency within the corresponding shaded artifact itself.